spec/audited/auditor_spec.rb

```diff
@@ -604,4 +604,15 @@ def non_column_attr=(val)
       }.to_not change( Audited.audit_class, :count )
     end
   end
+
+  describe "Update only for authenticated user" do
+    before do
+      Audited.only_authenticated_user = true
```

Comment:
Also add an `after` block to reset this to the default value (`false`) to prevent it interfering with later tests.